### PR TITLE
Fetch CELO nfts from tatum api client

### PR DESCRIPTION
**Pull request by RubenSousaDinis**
## Summary

This PR adds the tatum api client to fetch celo nfts.

## Notion link

https://www.notion.so/talentprotocol/New-Profile-implementation-aee6a9f1073e4a0480b68d035c056177

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -13,7 +13,6 @@
 ActiveRecord::Schema.define(version: 2022_08_23_152927) do
 
   # These are extensions that must be enabled in order to support this database
-  enable_extension "pg_stat_statements"
```
Comment (verdefred) [inline db/schema.rb:16]: do we want to delete this?

File: lib/gnosis_chain_explorer/client.rb [hunk 2]
```diff
@@ -1,23 +0,0 @@
...
-    private
-
-    def headers
-      {
-        "Content-Type": "application/json",
-        Accept: "application/json"
-      }
-    end
-  end
-end
```
Comment (verdefred) [inline lib/gnosis_chain_explorer/client.rb:23]: do we want to delete this?

File: lib/web3/api_proxy.rb [hunk 3]
```diff
@@ -138,6 +138,29 @@ def moralis_nfts
...
+      tokens = response_body.map do |item|
+        item["metadata"].map do |token_metadata|
+          {
+            address: item["contractAddress"],
+            token_id: token_metadata["tokenId"],
+            amount: 1,
+            name: token_metadata["metadata"].present? ? token_metadata["metadata"]["name"] : nil,
+            symbol: token_metadata["metadata"].present? ? token_metadata["metadata"]["name"] : nil,
+            token_uri: token_metadata["url"].start_with?("http") ? token_metadata["url"] : nil,
+            metadata: token_metadata["metadata"].present? ? token_metadata["metadata"] : nil
```
Comment (verdefred) [inline lib/web3/api_proxy.rb:156]: is there a reason to get all these things? I think we only need the address and the token id to then get the nft image, right?

**Review by verdefred — COMMENTED** [review]
great work, just left a couple of questions

File: lib/gnosis_chain_explorer/client.rb (hunk 2 above)
Comment (RubenSousaDinis) [reply lib/gnosis_chain_explorer/client.rb:23]: this is now in the web3 folder

File: lib/web3/api_proxy.rb (hunk 3 above)
Comment (RubenSousaDinis) [reply lib/web3/api_proxy.rb:156]: I think it doesn't hurt to get as much data as we can. Might be good for debugging

File: db/schema.rb (hunk 1 above)
Comment (RubenSousaDinis) [reply db/schema.rb]: added back

File: lib/gnosis_chain_explorer/client.rb (hunk 2 above)
Comment (verdefred) [reply lib/gnosis_chain_explorer/client.rb:23]: but was it duplicated in 2 places? I don't see it being added anywhere in this PR

Comment (RubenSousaDinis) [reply lib/gnosis_chain_explorer/client.rb:23]: Yes, it was probably a rebase issue. It was already in the web3 folder

**Review by verdefred — APPROVED** [review]
🚀

**Pull request merged**